bot.py

```diff
@@ -116,16 +118,27 @@ async def play(self, message, command_content):
 
         audio_url = video_metadata.getbestaudio().url
 
-        if(self.voice_client is None):
+        if self.voice_client is None:
             self.voice_client = await message.author.voice.channel.connect()
 
-        if(self.voice_client.is_playing()):
+        if self.voice_client.is_playing():
             self.voice_client.stop()
 
         audio_source = discord.FFmpegPCMAudio(audio_url)
         self.voice_client.play(audio_source)
         await asyncio.sleep(10)
```

Comment:
Is this needed?

Reply:
If you try doing voice_client.play() while voice_client.is_playing() is true, you get an error. Since there is no song queue for now, this is need in order for the new audio stream to start playing without throwing an error.

Reply:
Seems fine for now, we'll just need to remember to remove it later.